main.go

```diff
@@ -82,6 +95,13 @@ func main() {
 		os.Exit(1)
 	}
 
+	// Enable profiling endpoint for authorised users
+	if cfg.EnableProfiler {
+		identityHandler := identity.Handler(cfg.ZebedeeURL)
```

Comment:
As we already have a Zebedee client, I think we should reuse it here:
```
		idCli := identity.NewAPIClient(<zebedee client>, cfg.ZebedeeURL)
		identityHandler := identityMiddleware.HandlerForHTTPClient(idCli)
...
```

Reply:
The NewAPIClient function is actually wants an rchttp.Clienter this is not accessible in the zebedee client as it is not exported. I would need to change dp-api-clients-go to allow access to the clienter

Reply:
ah ok, In the other service I used the Zebedee health client:
```
if cfg.EnableProfiler {
		log.Event(ctx, "creating identity client, as profiler is enabled", log.INFO)
		clients.ZebedeeHealth = health.NewClient("Zebedee", cfg.ZebedeeURL)
	}
```
...
```
if cfg.EnableProfiler {
		idCli := identity.NewAPIClient(clients.ZebedeeHealth.Client, cfg.ZebedeeURL)
		identityHandler := identityMiddleware.HandlerForHTTPClient(idCli)
		middlewareChain := alice.New(identityHandler).Then(http.DefaultServeMux)
		r.PathPrefix("/debug/").Handler(middlewareChain)
	}
```
But here we need the actual Zebedee client. Maybe I can just approve it, as this endpoint is only enabled in debugging mode?